client/src/components/NavBottom.js

```diff
@@ -65,7 +65,9 @@ const NavBottom = () => {
           </SVG>
         </NavLink>
       </NavItemLeft>
-      <NavItemRight active={location.pathname === '/household'}>
+      <NavItemRight
+        active={location.pathname === '/household' || '/displayHousehold'}
```

Comment:
THere is no route like `/displayHousehold`.

In addition, active will be `'/displayHousehold'` if the pathname is not `/household`. I think you tried to do something like this:

`active={location.pathname === '/household' || location.pathname === '/displayHousehold'}`

Reply:
Thank you!